docker-compose.yml

```diff
@@ -47,15 +47,16 @@ services:
     image: yalelibraryit/dc-blacklight:${BLACKLIGHT_VERSION}
     command: bash -c "echo $$(date -u +%FT%TZ) > DEPLOYED_AT && /sbin/my_init"
     environment:
-      POSTGRES_DB: blacklight_yul_development
```

Comment:
you don't need to tell blacklight the name of its database?

Reply:
We're relying on rails convention to supply the name instead of configuration.  I.E. if we don't specify the names anywhere, rails has a set of conventions for naming databases in each environment that we can rely on.  Otherwise, we need to correctly specify the name in multiple config and compose files.  This change came out of one of those places having the wrong name specified.